bin/pycbc_source_probability_offline

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+"""
+Compute source probabilities using mchirp estimation method for all events
+in a chunk with an IFAR above certain threshold.
+"""
+import os
+import argparse
+import h5py, logging
+import numpy as np
+import json
+import tqdm
+from pycbc.io import hdf
+from pycbc.pnutils import mass1_mass2_to_mchirp_eta
+from pycbc import mchirp_area
+import pycbc.version
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--trigger-file')
+parser.add_argument('--bank-file')
+parser.add_argument('--single-detector-triggers', nargs='+', default=None)
+parser.add_argument('--search-tag',
+                    help='eg: PYCBC_AllSky, PYCBC_HighMass')
+parser.add_argument('--ifar-threshold', type=float, default=None,
+                    help='Select only candidate events with IFAR '
+                         'above threshold.')
+parser.add_argument('--verbose', action='count')
+parser.add_argument("--version", action="version",
+                    version=pycbc.version.git_verbose_msg)
+mchirp_area.insert_args(parser)
+args = parser.parse_args()
+
+mc_area_args = mchirp_area.from_cli(args)
```

Comment:
maybe you want to check here (ie `assert`) that the NS/mass gap boundaries are the same number, otherwise if they are not then p(MG) could be nonzero and then we would get incorrect results by removing p(MG)

Reply:
Thanks for bringing this up. I thought also on the possibility of having an extra argument like '--include-mass-gap' in case anyone wants the full set (?)

Reply:
ok - that would be more complicated than we want to do now in this script. maybe to allow for this in future you could define a variable `include_mass_gap` which is set to False and then have this check performed only if the variable is False and also remove the MG probability only in this case. 
.. i.e. this would allow for the option to be added later.